File: Exercícios sobre POO/Carro 1-2-6/Carro em python/motor.py
class Motor:
    def __init__(self, tipo, potencia) -> None:
        self.tipo = tipo
        self.potencia = potencia
        self.ligado = False
    
    def ligar(self):
        self.ligado = True
        print("O motor esta ligado")

    def desligar(self):
        self.ligado = False
        print("O motor esta desligado")

File: Exercícios sobre POO/Carro 1-2-6/Carro em java/src/Motor.java
public class Motor {
    private String tipo;
    private double potencia;
    private boolean ligado;

    public Motor(String tipo, int potencia) {
        this.tipo = tipo;
        this.potencia = potencia;
    }

    public boolean get_ligado(){
        return this.ligado;
    }

    public void ligar(){
        this.ligado = true;
        System.out.println("O motor está ligado");
    }
    public void desligar(){
        this.ligado = false;
        System.out.println("O motor está desligado");
    }
}